cpython/Makefile

```diff
@@ -102,6 +102,7 @@ $(BUILD)/$(LIB): $(BUILD)/Makefile $(BUILD)/pyconfig.h $(BUILD)/Modules/Setup.lo
 	cp Setup.local $(BUILD)/Modules/
 	( \
 		cd $(BUILD); \
+		make regen-frozen; \
```

Comment:
I had trouble with this before because it builds and runs an executable but it has already been configured to cross compile. But if this just works, that's convenient.